src/reducers/load.js

```diff
@@ -0,0 +1,15 @@
+import { COMPLETE_LOAD } from '../actions/load.js';
+
+const load = (state = {}, action) => {
+  switch (action.type) {
+    case COMPLETE_LOAD:
+      return {
+        ...state,
+        complete: true
```

Comment:
I can't find this used anywhere (but it's prolly somewhere not in the diff). Can you point me to where this is used, please?

Reply:
shop-app - https://github.com/Polymer/shop/blob/lit-element/src/components/shop-app.js#L351

Reply:
The `load` redux state has been removed - by moving the dynamic import() from the action to the shell component, we no longer need this redux state (can just be tracked in the shell component state).